designer/client/reducers/component/componentReducer.validations.ts

```diff
@@ -43,7 +43,20 @@ const validateContent = ({ content }) => {
   return errors;
 };
 
-const validateList = (component) => {};
+const validateList = (component) => {
+  const errors: any = {};
+  if (
+    !component.values ||
+    !component.values.list ||
+    component.values.list === "-1"
+  ) {
```

Comment:
you can use optional chaining + nullish coalesce to clean this up
```suggestion
  if ( component?.values?.list ?? false ) 
```
so if component.values is nullish (undefined or null), treat `component?.values?.list` to be false. 

you can do this also, `component?.values?.list === "-1"`